waddrmgr/manager.go

```diff
@@ -2289,26 +2289,26 @@ func Open(namespace walletdb.Namespace, pubPassphrase []byte, chainParams *chain
 //
 // A ManagerError with an error code of ErrAlreadyExists will be returned the
 // address manager already exists in the specified namespace.
-func Create(namespace walletdb.Namespace, seed, pubPassphrase, privPassphrase []byte, chainParams *chaincfg.Params, config *ScryptOptions) (*Manager, error) {
+func Create(namespace walletdb.Namespace, seed, pubPassphrase, privPassphrase []byte, chainParams *chaincfg.Params, config *ScryptOptions) error {
```

Comment:
But if your calling code (unlike how the wallet package now works) actually does end up using the opened manager from the Create func, you can't use defer.  It should only defer when there was an error.  The only "sane" way to do that is to use named returns and check the returned err in a defer before deciding whether to close or not, and I know how much you love named returns.